src/borough/borough.repository.ts

```diff
@@ -101,11 +101,13 @@ export class BoroughRepository {
     communityDistrictId,
     limit,
     offset,
+    managingAgency,
   }: {
     boroughId: string;
     communityDistrictId: string;
     limit: number;
     offset: number;
+    managingAgency: string;
```

Comment:
You're already checking for truthiness underneath, so the code itself is good. But for type housekeeping, I think `managingAgency` could also be `undefined`.

```suggestion
    managingAgency: string | undefined;
```